### PR TITLE
Added PUT Load tests

### PR DESCRIPTION
**Pull request by SkandaSrinivasan**
Hi I am done with the loadtests for PUT. Errors start at a rate of 8300 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,28 @@ Status Codes  [code:count]               200:699975
...
+```
+Requests      [total, rate, throughput]         800000, 8000.01, 7999.98
+Duration      [total, attack, wait]             1m40s, 1m40s, 354.748µs
+Latencies     [min, mean, 50, 90, 95, 99, max]  129.992µs, 497.678µs, 205.491µs, 293.266µs, 458.709µs, 3.232ms, 149.632ms
+Bytes In      [total, mean]                     0, 0.00
+Bytes Out     [total, mean]                     15200000, 19.00
+Success       [ratio]                           100.00%
+Status Codes  [code:count]                      201:800000
+```
+![PUT-Benchmark](benchmarks/put-loadtest-100s.png)
```
Comment (burntcarrot) [inline README.md:228]: Could you please remove the old tests (line 166-181) and the old device specifications (line 154-162)?

**Review by burntcarrot — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SkandaSrinivasan) [reply README.md]: Hi @burntcarrot , I have removed the old tests and related info. Let me know if anything else is needed.

Comment (burntcarrot) [reply README.md]: Thanks for the changes!

**Comment by burntcarrot** [comment]
@SkandaSrinivasan Thank you for your contribution!

**Pull request merged**